src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.PWMSparkMax;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.subsystems.UnitModel;
+
+/**
+ * this subsystem takes the balls from the field to the funnel.
+ */
+public class Intake extends SubsystemBase {
+    private PWMSparkMax motor = new PWMSparkMax(Ports.Intake.MOTOR);

```

Comment:
You should use `CANSparkMax`

Reply:
In order to so, he need to add some files to build.gradle and a vendordeps json file, so he can't do it right now, i will add it later for him.

Reply:
I updated it, now @natymeitav you can use the `CANSparkMax` (Remember, it's a brushless motor)